src/ui.data-table/UTable.vue

```diff
@@ -201,6 +201,9 @@
               >
                 <slot :name="`cell-${key}`" :value="slotValues.value" :row="slotValues.row" />
               </template>
+              <template #nested-row="{ row: nestedRow }">
```

Comment:
its not a `nestedRow`, it's `parentRow`, and I don't see where you pass real data into row. Please test in in story to be ensure that data is passing correctly.